moreDomManipulation_HW/app.js

```diff
@@ -0,0 +1,85 @@
+console.log(document.body.style.fontFamily, 'here')
+document.body.style.fontFamily = "Arial, sans-serif";
+console.log(document.body.style.fontFamily)
+document.querySelector('span#nickname').innerText = 'Timon';
+document.querySelector('span#favorite').innerText = 'Spider';
+document.querySelector('#hometown').innerText = 'El Paso';
+const listitem = document.querySelectorAll('li');
+
+for (i of listitem) {
+    i.classList.add("listitem");
+}
+console.log(listitem);
+document.querySelectorAll('.listitem');
+for (i of listitem) {
+    i.style.color = "rebeccapurple";
```

Comment:
Nice work of using a for-of loop.
you can move this line of code and but it in the previous for-of loop and save some time.
```suggestion
  
```